config/initializers/paperclip.rb

```diff
@@ -10,6 +10,11 @@
   Pageflow::PaperclipInterpolations::Support.class_basename(attachment, style)
 end
 
+Paperclip.interpolates(:attachments_path_name) do |attachment, _style|
+  record = attachment.instance
+  record.respond_to?(:attachments_path_name) ? record.attachments_path_name : 'attachment_on_s3s'
```

Comment:
Hard coding the name does not work. There are other attachments (e.g. video posters) that need to use their proper name for this interpolation. Since [`interpolates` merely defines a method](https://github.com/thoughtbot/paperclip/blob/17050e41aa31613da23c7c1b56381f431ee9d3ff/lib/paperclip/interpolations.rb#L13) on `Paperclip::Interpolations`, we should be able to call the original `attachment` method when we rename the `attachment` param to prevent shadowing.

Reply:
Fixed in 039b0208615dfa81c26b592860db9b541438ee9b